core/deployment/src/main/java/io/quarkus/deployment/steps/BannerProcessor.java

```diff
@@ -114,9 +113,10 @@ private boolean isQuarkusCoreBanner(URL url) throws IOException {
         // We determine whether the banner is the default by checking to see if the jar that contains it also
         // contains this class. This way although somewhat complicated guarantees that any rename of artifacts
         // won't affect the check
-        try (JarFile jarFile = new JarFile(Paths.get(new URI(jarPath)).toFile())) {
+        try {
+            JarFile jarFile = new JarFile(new File(jarPath));
```

Comment:
The `jarFile` needs to be closed. Was there a reason to move it inside the `try`?

Reply:
no, you are right @aloubyansky reverted 👍  good catch!